src/components/form/dt-location-map/dt-location-map-item.js

```diff
@@ -162,32 +162,45 @@ export default class DtLocationMapItem extends LitElement {
         }
 
         .field-container {
+          --dt-location-map-border-radius: var(--dt-form-border-radius, 10px);
           display: flex;
           margin-bottom: 0.5rem;
         }
+        .field-container > *:first-child {
+          border-start-start-radius: var(--dt-location-map-border-radius, 0);
```

Comment:
The default/fallback on these is unnecessary on these since you are defining a definite value on line 165 above with default value. To avoid confusion, I'd remove the default values on these.